frontend/src/api/Environment/getEnvironment.js

```diff
@@ -49,10 +48,6 @@ const getEnvironment = ({ environmentUri }) => ({
           outputs
           resources
         }
-        dashboardsEnabled
```

Comment:
Shouldn't it be deleted in the other PRs? It seems that it will break editing the env since the other parameters are not migrated

Reply:
The issue is that it is duplicated, so I just removed one `dashboardsEnabled`